src/commands/synthetics/__tests__/reporters/default.test.ts

```diff
@@ -160,6 +166,7 @@ describe('Default reporter', () => {
     const baseSummary: Summary = createSummary()
 
     const complexSummary: Summary = {
+      batchId: 'batch-id',
```

Comment:
We should set the `batchId` to the `baseSummary` instead:
```ts
const baseSummary = {...createSummary(), batchId: 'batch-id'}

const complexSummary: Summary = {
  ...baseSummary,
  criticalErrors: 2,
  ...
}
```
 Right now, the 2 other cases ("simple case" + "case where [...] empty") do not render the `Results URL: ` line 🤔

Reply:
That was done willingly to see if both cases are handled without issue